### PR TITLE
CYB #79 File DropZone integration tests

### PR DESCRIPTION
**Pull request by artemmakogon**
Please review.

### CODE REVIEW

**Comment by artemmakogon** [comment]
Please review only changes related to File DropZone integration tests: 
CYB #79: add integration tests for DropZone

File: integration-tests/file-drop-zone-integration-tests/pom.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright © 2014 Cask Data, Inc.
```
Comment (awholegunch) [inline integration-tests/file-drop-zone-integration-tests/pom.xml]: Remove "~" characters

**Comment by sreevatsanraman** [comment]
Why does this have file-drop-zone commits? I have merged file drop zone and this should only have the integration test.

**Comment by sreevatsanraman** [comment]
@artemmakogon @ashvayka - please re-create this PR with only integration test changes.

**Pull request closed without merging**